.github/workflows/scripts/lnproxy-sync.js

```diff
@@ -0,0 +1,37 @@
+const fs = require('fs');
+
+let rawRelays = JSON.parse(fs.readFileSync('./lnproxy_relays.json'));
+let formattedRelays = [];
+
+let torCount = 0;
+let i2pCount = 0;
+let clearnetCount = 0;
+
+for (let url of rawRelays) {
+  let relayType;
+  const LNPROXY_API_PATH = '/spec'
+  const fqdn = url.replace(LNPROXY_API_PATH, '');
+  if (fqdn.endsWith('.onion')) {
+    relayType = "TOR";
+    torCount++;
+  }
+  else if (fqdn.endsWith('i2p')) {
+    relayType = "I2P";
+    i2pCount++;
+  }
+  else {
+    relayType = "Clearnet";
+    clearnetCount++;
+  }
+
+  let relayName = `LNProxy ${relayType}${relayType === "TOR" ? torCount : ''}${relayType === "I2P" ? i2pCount : ''}${relayType === "Clearnet" ? clearnetCount : ''}`;
```

Comment:
What about using the first 6 characters of the domain name ? e.g.:
```javascript
let relayName = ${relayType}${relayType === "TOR" ? torCount : ''}${relayType === "I2P" ? i2pCount : ''}${relayType === "Clearnet" ? clearnetCount : ''} ${url.split('/')[2].substring(0,6)}
```

So it would read: **TOR1 w3sqmn** I think something in this line will make the dropdown menu more transparent for the user (rather than just a number, as the user does not know what is behind the number)

Reply:
Yeah, this could go in many ways, there's probably more you could inspect on a relay other than just the name. For example status - some kind of ping to show whether the host is up (currently one of the tor lnproxy relays is down).

But first six is a good start :)

Reply:
We will just need to keep an eye on the relay list - something like "https://www.paidrelayproxiesfromthefuture.com" would render as "Clearnet1 www.pa"